backend/inspirehep/matcher/api.py

```diff
@@ -159,7 +167,7 @@ def match_reference_control_numbers(reference):
         except KeyError:
             return None
 
-    configs = match_reference_config(reference)
+    configs = match_reference_config(reference, use_relaxed_titles_matching=True)
```

Comment:
I think it would be cleaner to expose this fact to the top-level API functions. Here you're using the fact that `match_reference_control_numbers` is used only in the search API, while the regular `match_reference` is used elsewhere. But that's incidental and not clear from the API. Maybe rename this function to`match_reference_control_numbers_with_relaxed_journal_titles` to be explicit?